blocks/carousel/carousel.css

```diff
@@ -29,16 +36,6 @@
     transition: all 0.5s;
 }
 
-.carousel .carousel-slide > div:nth-child(1) {
```

Comment:
I see you've removed some stuff from carousel.css is that for Leadspace block specifically or just some other fixes to carousel that are just in the same PR?